feature-libs/quote/core/connectors/quote.connector.ts

```diff
@@ -6,73 +6,124 @@
 
 import { inject, Injectable } from '@angular/core';
 import { PaginationModel } from '@spartacus/core';
-import { Observable } from 'rxjs';
 import {
+  Comment,
   Quote,
   QuoteActionType,
   QuoteDiscount,
   QuoteList,
   QuoteMetadata,
   QuoteStarter,
-  Comment,
 } from '@spartacus/quote/root';
+import { Observable } from 'rxjs';
 import { QuoteAdapter } from './quote.adapter';
 
 @Injectable()
 export class QuoteConnector {
   protected quoteAdapter = inject(QuoteAdapter);
 
-  public getQuotes(
+  /**
+   * Fetches a list of quotes according to the pagination provided
+   * @param userId Quote user
+   * @param pagination Pagination attributes
+   * @returns Observable emitting a quote list
+   */
+  getQuotes(
     userId: string,
     pagination: PaginationModel
   ): Observable<QuoteList> {
     return this.quoteAdapter.getQuotes(userId, pagination);
   }
 
-  public createQuote(
-    userId: string,
-    quoteStarter: QuoteStarter
-  ): Observable<Quote> {
+  /**
+   * Creates a quote
+   * @param userId Quote user
+   * @param quoteStarter Attributes needed to create a quote
+   * @returns Observable emitting a quote
+   */
+  createQuote(userId: string, quoteStarter: QuoteStarter): Observable<Quote> {
     return this.quoteAdapter.createQuote(userId, quoteStarter);
   }
 
-  public getQuote(userId: string, quoteCode: string): Observable<Quote> {
+  /**
+   * Gets a quote specified by its ID (aka code)
```

Comment:
Could we use `retrieves` got **getter** instead of `fetches / gets` ...?

Reply:
Changed to 'fetches'